geocity/apps/submissions/tables.py

```diff
@@ -361,3 +374,80 @@ class Meta:
             "status",
         )
         template_name = "django_tables2/bootstrap.html"
+
+
+class PandasExportMixin(ExportMixin):
+    def create_export(self, export_format):
+        advanced = self.request.GET.get("_advanced", False)
+
+        if not export_format in ["xlsx"]:
+            raise NotImplementedError
+
+        if not advanced:
+            return super().create_export(export_format)
+
+        # Retrieve entities associated to the user
+        entities = AdministrativeEntity.objects.associated_to_user(self.request.user)
+
+        # Take all submission except status draft
+        submissions_qs = self.get_pandas_table_data().filter(
+            Q(administrative_entity__in=entities),
+            ~Q(status=models.Submission.STATUS_DRAFT),
+        )
+
+        if not has_permission_to_amend_submission(
+            self.request.user, submissions_qs.first()
```

Comment:
`is_backoffice_of_entity` use this instead ? Does the submission really need to be amendable ? @monodo